listenbrainz/webserver/views/atom.py

```diff
@@ -0,0 +1,29 @@
+from feedgen.feed import FeedGenerator
```

Comment:
Nice to have a test! However, I think this should be moved to an actual python test, rather than a function in the views.